assignments/array-methods.js

```diff
@@ -56,28 +56,48 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach((el, idx) => {
+    fullName.push(`${el.first_name} ${el.last_name}`);
+    })
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+allCaps = runners.map((r) => r.first_name.toUpperCase())
```

Comment:
Very Clean code Karl!!! 
Just to note, with es6 Arrow Function Syntax, if you only have ONE parameter coming into the function, you don't NEED to put parenthesis around it! .. your code COULD look like this:
allCaps = runners.map(r => r.first_name.toUpperCase())